src/utility/VersionUtility.ts

```diff
@@ -48,25 +48,17 @@ export class VersionUtility {
     }
 
     // Returns the CLI version that this VSCode session expects
-    public static async getExpectedCliVersionAsync(context: vscode.ExtensionContext, experimentationService: IExperimentationService, packageJsonContent: object, logger: Logger): Promise<string> {
-        const bridgeEnvironment = EnvironmentUtility.getBridgeEnvironment(logger);
-
-        // This call will return the upcoming version for treatment group; control & everyone else will get the stable prod version. Undefined if the flight is currently inactive.
-        // TODO(ansoedal): Disable ExP permanently. For now this call will return undefined, as long as the treatment variable doesn't exist
-        const expBinariesVersion = experimentationService.getTreatmentVariable<string>(/*configId*/ `vscode`, /*name*/ `mindaroBinariesVersion-doesnotexist`);
-        logger.trace(`Call to ExP returned version '${expBinariesVersion}'`);
+    public static async getExpectedCliVersionAsync(packageJsonContent: object): Promise<string> {
+        const bridgeEnvironment = EnvironmentUtility.getBridgeEnvironment();
 
         // How we resolve the expected CLI version:
         //  1. Is the BRIDGE_CLI_VERSION environment variable set? Use that one.
         //  2. Is a flight in progress? Get the binaries version from ExP.
         //  3. Fall back onto whichever LKS version was specified in package.json. (For dev and staging, this is whichever version matches this extension. For Prod, this is a variable we specify in the Mindaro-Connect-VSCode pipeline.)
         const packageJsonVersion: string = packageJsonContent[`extensionMetadata`][`expectedCLIVersion`];
         const expectedCLIVersion = process.env.BRIDGE_CLI_VERSION != null ? process.env.BRIDGE_CLI_VERSION :
-            (expBinariesVersion != null && bridgeEnvironment === Environment.Production ? expBinariesVersion :
-            (process.env.BRIDGE_BUILD_PATH != null ? null :
-            (packageJsonVersion != null && !packageJsonVersion.startsWith(`$`) ? packageJsonVersion : null)));
+            (packageJsonVersion != null && !packageJsonVersion.startsWith(`$`) ? packageJsonVersion : null);
 
-        logger.trace(`Resolved expected CLI version '${expectedCLIVersion}'`);
```

Comment:
That's likely a matter of personal preference, so feel free to ignore: this log was useful in the past to remove ambiguity on what version of the binaries were presently being used by VS Code. If the issue is just with providing the logger when in a unit-test context, then maybe we could create a mock logger (I believe we do have examples of mocks we used already in the code)?
In any case, the logger is used pretty much everywhere in the codebase, so I expect that you will need this mock at some point if you add more unit-tests.

That said, again that's a fairly minor comment, so please feel free to keep as it is if you prefer the version without a logger.